daal4py/sklearn/test/test_run_to_run_stability_tests.py

```diff
@@ -14,322 +14,350 @@
-        run_clf_test(run_svc_rbf)
-
-    def test_svc_rbf_const_gamma(self):
-        run_clf_test(run_svc_rbf_const_gamma)
-
-    #---------------------Failed---------------------
-
-    @unittest.skip("unstable")
-    def test_kmeans_plus_plus(self):
-        run_clf_test(run_kmeans_plus_plus)
-
-    @unittest.skip("unstable")
-    def test_kmeans_random(self):
-        run_clf_test(run_kmeans_random)
-
-    @unittest.skip("unstable")
-    def test_elasticnet_regression(self):
-        run_reg_test(run_elasticnet_regression)
-
-    @unittest.skip("unstable")
-    def test_lasso_regression(self):
-        run_reg_test(run_lasso_regression)
-
-    @unittest.skip("unstable")
-    def test_pca_full(self):
-        run_clf_test(run_pca_full)
-
-    @unittest.skip("unstable")
-    def test_pca_daal4py_corr(self):
-        run_clf_test(run_pca_daal4py_corr)
-
-    #---------------------Expected to be fixed in next release---------------------
-
-    @unittest.skip("unstable")
-    def test_log_regression_newton(self):
-        run_clf_test(run_log_regression_newton)
-
-    @unittest.skip("unstable")
-    def test_log_regression_lbfgs(self):
-        run_clf_test(run_log_regression_lbfgs)
-
-    @unittest.skip("unstable")
-    def test_log_regression_cv_newton(self):
-        run_clf_test(run_log_regression_cv_newton)
-
-    @unittest.skip("unstable")
-    def test_log_regression_cv_lbfgs(self):
-        run_clf_test(run_log_regression_cv_lbfgs)
-
-    @unittest.skip("unstable")
-    def test_linear_regression(self):
-        run_reg_test(run_linear_regression)
-
-    @unittest.skip("unstable")
-    def test_ridge_regression(self):
-        run_reg_test(run_ridge_regression)
-
-    @unittest.skip("unstable")
-    def test_rf_class(self):
-        run_clf_test(run_rf_class)
-
-    @unittest.skip("unstable")
-    def test_rf_reg(self):
-        run_reg_test(run_rf_regression)
-
 
-if __name__ == '__main__':
-    unittest.main()
+def _run_test(model, methods, dataset):
+    # ------------light datasets------------
+    if dataset in ['blobs', 'classifier', 'sparse']:
+        X, y = load_iris(return_X_y=True)
+        if dataset == 'sparse':
+            X = sparse.csr_matrix(X)
+    elif dataset == 'regression':
+        X, y = load_boston(return_X_y=True)
+    else:
+        raise ValueError('Unknown dataset type')
+
+    baseline, name = func(X, y, model, methods)
+
+    for i in range(10):
+        res, _ = func(X, y, model, methods)
+
+        for a, b, n in zip(res, baseline, name):
+            np.testing.assert_allclose(a, b, rtol=0.0, atol=0.0,
+                                       err_msg=str(n + " is incorrect"))
+    # ------------hard datasets------------
+    if dataset in ['blobs', 'classifier', 'sparse']:
+        X, y = load_breast_cancer(return_X_y=True)
+        if dataset == 'sparse':
+            X = sparse.csr_matrix(X)
+    elif dataset == 'regression':
+        X, y = make_regression(n_samples=5000, n_features=100,

```

Comment:
I would to propose to use dataset from scikit-leartn, for example [diabet](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_diabetes.html#sklearn.datasets.load_diabetes) because we will add SVR, where there may be the same problem as for the classification in the case of generate data.